README.md

```diff
@@ -9,13 +9,14 @@ The main entrypoint (or homepage) is `index.md` and new pages must be referenced
 
 ### To test locally
 
-Making sure [Ruby](https://www.ruby-lang.org/en/documentation/installation/) and [Bundler](https://bundler.io/) are installed in your machine, run the following:
+Making sure [Ruby](https://www.ruby-lang.org/en/documentation/installation/) `2.7` and [Bundler](https://bundler.io/) are installed in your machine, run the following:
```

Comment:
I was having this because i installed ruby 3, GH pages doesn't support ruby 3 :[ 

https://talk.jekyllrb.com/t/error-no-implicit-conversion-of-hash-into-integer/5890

Reply:
Nice catch!